packages/dev/core/src/Misc/fileTools.ts

```diff
@@ -175,8 +177,8 @@ export const LoadImage = (
     mimeType: string = "",
     imageBitmapOptions?: ImageBitmapOptions
 ): Nullable<HTMLImageElement> => {
-    if (typeof HTMLImageElement === "undefined") {
-        onError("Displacement map is only supported in web environment.");
+    if (typeof HTMLImageElement === "undefined" && typeof _native === "undefined") {
+        onError("Displacement map is only supported in web or BabylonNative environments.");

```

Comment:
I don't understand this error. What does displacement map have to do with `HTMLImageElement`?